sites/partners/src/listings/PaperListingForm/sections/Units.tsx

```diff
@@ -46,13 +53,28 @@ const FormUnits = ({ units, setUnits, disableUnitsAccordion }: UnitProps) => {
     setValue("disableUnitsAccordion", disableUnitsAccordion ? "true" : "false")
   }, [disableUnitsAccordion, setValue])
 
+  useEffect(() => {
+    if (unitsSummary !== undefined) {
+      setUnitsSummarized(
+        groupNonReservedAndReservedSummaries(
+          unitsSummary.byNonReservedUnitType,
+          unitsSummary.byReservedType
+        )
+      )
+    }
+  }, [setUnitsSummarized])
```

Comment:
So this sets the initial unitsSummarized once the page loads, right? If there is no unitsSummary passed in as a prop will the table just be blank when you select that option?

Reply:
unitsSummary is a required prop, so in theory it should always be provided. But the state var defaults to an empty list, so if for some reason unitsSummary is undefined, yes we should just see an empty table.